### PR TITLE
v3

### PR DESCRIPTION
**Pull request by Defelo**

### CODE REVIEW

File: morpheushelper/cogs/custom/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from . import translations as _
```
Comment (ghost) [inline morpheushelper/cogs/custom/__init__.py]: do you need this import there?

File: morpheushelper/morpheushelper.py [hunk 2]
```diff
@@ -1,360 +1,36 @@
...
-        "\033[1m\033[31m%s Cog%s disabled:\033[0m", len(disabled_cogs), "s" if len(disabled_cogs) > 1 else ""
-    )
-    for name in disabled_cogs:
-        root_logger.info(" - %s", name)
+if SENTRY_DSN:
+    logger.debug("initializing sentry")
+    setup_sentry(SENTRY_DSN, Config.NAME, Config.VERSION)
 
-bot.run(os.environ["TOKEN"])
+__import__("bot").run()
```
Comment (ghost) [inline morpheushelper/morpheushelper.py]: How about if u make the import at the top level and execute it at the end

File: morpheushelper/cogs/custom/__init__.py (hunk 1 above)
Comment (Defelo) [reply morpheushelper/cogs/custom/__init__.py]: yes, this is needed to load the translation namespaces

File: morpheushelper/morpheushelper.py (hunk 2 above)
Comment (Defelo) [reply morpheushelper/morpheushelper.py]: this is not possible because the config needs to be loaded before `bot` is imported

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by TheCataliasTNT2k — APPROVED** [review]
Yes, it is completed.... It was a loooong way....

**Pull request merged**